src/parser.ts

```diff
@@ -103,12 +103,64 @@ const getVideoCount = (channel: any) => {
  */
 const getSubscriberCount = (channel: any) => {
     try {
-        return channel.subscriberCountText.simpleText.split(' ').shift();
+        let count = channel.subscriberCountText.simpleText.split(' ').shift();
+        let lastChar = count.slice(-1);
+        if (parseInt(lastChar) === NaN) {
```

Comment:
This statement will almost never be true, when checking if something is NaN you may want to use `isNaN()`  
This error means subscriber counts like `200` or `16` were cut down to `20` and `1`
  
https://stackoverflow.com/questions/825402/why-does-isnan-string-with-spaces-equal-false

Reply:
I can fix this in my branch but I figured I'd let you know either way

Reply:
My bad. Never been into this error. Don't know how that function worked fine when i tested it though. 
Also, thanks for pointing that out😉.